src/github/client/github-queries.ts

```diff
@@ -361,12 +362,50 @@ export type getDeploymentsResponse = {
 	}
 };
 
+export const getDeploymentsQueryByCreatedAtDesc = `query ($owner: String!, $repo: String!, $per_page: Int!, $cursor: String) {
+  repository(owner: $owner, name: $repo){
+    deployments(first: $per_page, after: $cursor, orderBy: { direction: DESC, field: CREATED_AT }) {
```

Comment:
**IMPORTANT**
This is interesting. The current order of the graphql seems to be ASC. 
However, in order to make the `fromDate` work, I'll have to change it to DESC. 

The problem might be it will affect existing customers. I think let's have a chat on how to deal with it.

Reply:
So I added an admin api to reset the cursor to null when the `deploymentStatus` is null or failed or pending.